test/tests/functional/pbs_reservations.py

```diff
@@ -2156,3 +2156,28 @@ def test_standing_resv_resc_used(self):
 
         self.server.expect(RESV, 'queue', op=UNSET, id=rid, offset=10)
         self.server.expect(SERVER, {'resources_assigned.ncpus': 0})
+
+    def test_server_recover_resv_queue(self):
+        """
+        Test that PBS server can recover a reservation queue after a
+        restart
+        """
+
+        a = {'resources_available.ncpus': 1}
+        self.server.create_vnodes('vn', a, num=2, mom=self.mom)
+        now = int(time.time())
+        rid = self.submit_reservation(user=TEST_USER, select='1:ncpus=1',
```

Comment:
do we need to provide 1:ncpus=1?

Reply:
We don't have to. But since I'm using a helper function that has been used in other tests too, this function expects select, start, end and user to be provided